### PR TITLE
[DO NOT MERGE] Propose new sample format

### PR DESCRIPTION
**Pull request by alixhami**
Proposed template for auto generated Python samples.
See more details in the [Node.js PR](https://github.com/googleapis/nodejs-dlp/pull/154) description.

### CODE REVIEW

File: dlp/dlp_inspect_image_file.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# limitations under the License.
+import sys
+
+# [START dlp_inspect_text_file]
+# Import the Google Cloud Data Loss Prevention library
+import google.cloud.dlp
+
+
+def inspect_image_file(project_id='YOUR_PROJECT_ID',
+                       filepath='path/to/image.png'):
```
Comment (theacodes) [inline dlp/dlp_inspect_image_file.py]: No hanging indents, please.

If you really wanna be clever run it through `black`.

File: dlp/dlp_inspect_image_file.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            print('Quote: {}'.format(finding.quote))
+            print('Info type: {}'.format(finding.info_type.name))
+            print('Likelihood: {}'.format(finding.likelihood))
+    else:
+        print('No findings.')
+# [END dlp_inspect_text_file]
+
+
+if __name__ == '__main__':
+    inspect_image_file(project_id=sys.argv[1], filepath=sys.argv[2])
```
Comment (theacodes) [inline dlp/dlp_inspect_image_file.py]: This is really less than ideal, as if the user tries to run without arguments this they will get a really unhelpful error.

File: dlp/dlp_inspect_image_file.py (hunk 1 above)
Comment (alixhami) [reply dlp/dlp_inspect_image_file.py]: Surprisingly, `black` wanted me to just have a long line that violates the 79 characters rule, so I didn't follow it. Fixed indentation in https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1756/commits/a8d248c9c3471ce9281ec6cecae2494b9cbce40c.

**Comment by engelke** [comment]
Please address requested changes if this PR is still desired.

**Comment by leahecole** [comment]
@alixhami Reopen if still relevant, otherwise, closing.

**Comment by meredithslota** [comment]
With the format guidelines updated in #2565 this feels obsolete. Ok to close?

**Comment by tmatsuo** [comment]
Let's close this. Feel free to reopen.

**Pull request closed without merging**